lib/index.js

```diff
@@ -1,3 +1,12 @@
+var Fake = require('./fake');
+var Helpers = require('./helpers');
+var Name = require('./name');
+var Random = require('./random');
+var Lorem = require('./lorem');
+var Address = require('./address');
+var Account = require('./account');
+var Phone = require('./phone_number');
+
 /**
```

Comment:
I think you'll walk me through this tomorrow because I can't see where the names we are requiring are used inside of `lib/index.js`. Are they somehow attached to self? 🤔